### PR TITLE
Added readme for tests 

### PR DESCRIPTION
**Pull request by OGKevin**
@dnl-blkv Whats the deal with this file: https://github.com/bunq/sdk_csharp/pull/2/files#diff-ea9c97899574dc6e5026dbf4e3a81fd5

Should it be removed ?

### CODE REVIEW

**Comment by dnl-blkv** [comment]
@OGKevin yes, remove that ApexVCS please. Looks weird :D

File: .idea/.idea.BunqSdkCsharp/.idea/preferred-vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="PreferredVcsStorage">
+    <preferredVcsName>ApexVCS</preferredVcsName>
+  </component>
+</project>
```
Comment (dnl-blkv) [inline .idea/.idea.BunqSdkCsharp/.idea/preferred-vcs.xml]: Remove this file please.

File: BunqSdk.Tests/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+are also tested :thumbs_up:.
+
+## Configuration
+To run the tests you must first setup the test configuration JSON. The example
+of a configuration file is located at [`Tests/BunqSdkCsharpTest/Resources/config.example.json`](./BunqSdkCsharpTest/Resources/config.example.json).
+In order to make use of the configuration file, please copy the example to the
+same directory, fill in your sandbox user data and rename the copy to `config.json`.
+
+## Execution
+To run tests via Rider, you can right click on the BunqSdk.Tests solution and should be able to run
```
Comment (dnl-blkv) [inline BunqSdk.Tests/README.md]: Please space after period!

File: README.md [hunk 3]
```diff
@@ -172,3 +172,7 @@ $ dotnet run ApiContextSaveSample.cs
 
 Please do not forget to set the `API_KEY` constant in `ApiContextSaveSample.cs` to your actual API key before running the
 sample!
+
+## Running tests
+Information regarding the test cases can be found in the [README.md](./BunqSdk.Tests/README.md)
+located in [test](./BunqSdk.Tests)
```
Comment (dnl-blkv) [inline README.md]: Please period in the end of the sentence :)

File: BunqSdk.Tests/README.md (hunk 2 above)
Comment (OGKevin) [inline BunqSdk.Tests/README.md]: But that is how the Solution/folder is called ? 😢

Comment (dnl-blkv) [inline BunqSdk.Tests/README.md]: Ahhh, ok, then `BunqSdk.Tests` 

I misread it indeed :D

Comment (dnl-blkv) [inline BunqSdk.Tests/README.md]: So, surround with code marks

File: BunqSdk.Tests/README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+are also tested :thumbs_up:.
+
+## Configuration
+To run the tests you must first setup the test configuration JSON. The example
+of a configuration file is located at [`Tests/BunqSdkCsharpTest/Resources/config.example.json`](./BunqSdkCsharpTest/Resources/config.example.json).
+In order to make use of the configuration file, please copy the example to the
+same directory, fill in your sandbox user data and rename the copy to `config.json`.
+
+## Execution
+To run tests via Rider, you can right click on the `BunqSdk.Tests solution and should be able to run
```
Comment (dnl-blkv) [inline BunqSdk.Tests/README.md]: Aaaaand the closing code marker is missing! :(

Comment (OGKevin) [inline BunqSdk.Tests/README.md]: 😕

**Pull request merged**